keyboards/jkdlab/binary_monkey/config.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 JKDLAB. <jkdlab.co@gmail.com>
+ * Copyright 2021 Jaehee <ljh34210329@gmail.com>
+ * 
+ * This program is free software: you can redistribute it and/or modify 
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+ 
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID       0x0324 
```

Comment:
This Vender ID is already in use, and needs to be changed. 

https://www.the-sz.com/products/usbid/index.php?v=0x0324&p=&n=

Reply:
Hi! I changed VENDOR_ID not used. Thank you!